src/components/ProjectCard/__snapshots__/ProjectCard.test.tsx.snap

```diff
@@ -26,19 +26,6 @@ exports[`<ProjectCard /> Default story renders snapshot 1`] = `
     <div
       class="project-card__meta"
     >
-      <svg
```

Comment:
let's keep a ProjectCard story with the icon as well as an example

Reply:
Renamed `Without Meta` story to `With Meta` and added `With Meta Icon` story as well